tests/schemas/test_fields.py

```diff
@@ -0,0 +1,50 @@
+import h5py
+import pytest
+from argschema import ArgSchemaParser, ArgSchema
+from marshmallow import ValidationError
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class H5InputTest(ArgSchema):
+    input_file = H5InputFile(required=True, description="test")
+
+
+@pytest.mark.parametrize(
+    "schema", [(H5InputTest)]
+)
+def test_h5_input_file_errors_not_exist(schema, tmp_path):
+    fp = tmp_path / "test.h5"
+    with pytest.raises(ValidationError) as e:
+        ArgSchemaParser(input_data={"input_file": str(fp)},
+                        schema_type=H5InputTest,
+                        args=[])
+    assert "OSError" in str(e.value)
```

Comment:
in a prior PR, Isaak showed me that you can use a `match` keyword with [raises](https://docs.pytest.org/en/latest/reference.html#pytest-raises).

Reply:
Do you want me to change it?

Reply:
just a nit and FYI, no strong feelings